commerce_coordinator/apps/demo_lms/views.py

```diff
@@ -83,3 +84,16 @@ def demo_purchase_complete(_):
     results = purchase_complete_signal.send_robust('demo_purchase_complete', order_results=callback_results)
 
     return _format_signal_send_response(results)
+
+
+def demo_get_sample_data(_):
+    """
+    Demonstrate a workflow that gets data from providers configured as pipeline steps.
+    """
+
+    sample_data = SampleDataRequested.run_filter()
+    return JsonResponse({"sample_data": sample_data})
+
+    # BJH: try using run_pipeline directly and see what else we get back
+    # results = SampleDataRequested.run_pipeline(sample_data=[])
+    # return JsonResponse(results)
```

Comment:
Given [this comment](https://github.com/edx/commerce-coordinator/pull/25/files#diff-f670ac19fec49481dce669f76686f569169a938818112ded8c8a1344d2910c7fR17) in the class definition `SampleDataRequested`, could the above be removed, given it seems we've settled on using `run_filter()` instead of `run_pipeline()`?

Reply:
Ah, yes, this was test code, I shoulda tagged it with my `NOCOMMIT` marker, I'll clean it up.